content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -290,12 +289,12 @@ Below you can find installation instructions for some popular Pod network plugin
 {{< tabs name="tabs-pod-install" >}}
 
 {{% tab name="Calico" %}}
-For more information about using Calico, see [Quickstart for Calico on Kubernetes](https://docs.projectcalico.org/latest/getting-started/kubernetes/), [Installing Calico for policy and networking](https://docs.projectcalico.org/latest/getting-started/kubernetes/installation/calico), and other related resources.
+[Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options designed to give you the most efficient networking across a range of situations, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
 
-For Calico to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init` or update the `calico.yml` file to match your Pod network. Note that Calico works on `amd64`, `arm64`, and `ppc64le` only.
+By default, Calico uses `192.168.0.0/16` to assign pod IP addresses, though this can be configured. For Calico to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init` or update the `calico.yaml` file to match your desired pod network CIDR. Calico works on several architectures, including `amd64`, `arm64`, and `ppc64le`.
```

Comment:
possibly :
> Calico works on several architectures, including `amd64`, `arm64`, and `ppc64le`.

can be moved to the above paragraph.

maybe the rest can be refactored to:

> By default, Calico uses `192.168.0.0/16` as the Pod network CIDR, though this can be configured in the the `calico.yaml` file. For Calico to work correctly, you need to pass this same CIDR to the `kubeadm init` command using the `--pod-network-cidr=192.168.0.0/16` flag or via the kubeadm configuration.